spec/models/employee_spec.rb

```diff
@@ -11,48 +11,56 @@
     it { should validate_presence_of(:private_number) }
   end
   let(:employee) { FactoryBot.create(:employee)  }
+  let(:employees) { FactoryBot.create_list(:employee, 10)  }
   let(:user) { User.create(email:'dumy2@example.com', password:'1234567', superadmin_role: true, user_role: false) }
   subject { Employee.new(name: 'Jose Estrada', email:'dumy@example.com', private_number: 4123, user_id: user.id) }
 
-  before :each do 
+  before :each do
     with_repeated_data = Employee.new(name: 'Juan', email:'dumy@example.com', private_number: 4123, user_id: user.id)
     with_repeated_data.save
   end
-  it 'name should be present' do 
-    subject.name = nil 
+  it 'name should be present' do
```

Comment:
Remember that you can add these validations like the below code
```
describe 'validations' do
    it { should validate_presence_of(:name) }
    it { should validate_presence_of(:email) }
  end
```

Reply:
Sure, even this sentences are exist in the top at file, already changed it, i deleted this three test to solved with this kinds validations syntax.

DONE